R-package/R/lgb.train.R

```diff
@@ -39,12 +39,11 @@
 #' valids <- list(test = dtest)
 #' model <- lgb.train(params,
 #'                    dtrain,
-#'                    100,
+#'                    10,
```

Comment:
@jameslamb Exactly! BTW, ideally, params for examples should be tuned, because currently we have awful training with huge number of 
```
[LightGBM] [Warning] No further splits with positive gain, best gain: -inf
[LightGBM] [Warning] Stopped training because there are no more leaves that meet the split requirements
```